int32 obj_id
int32 temporal_frequency
